integrationtests/tests/fixtures/mod.rs

```diff
@@ -169,16 +170,21 @@ pub async fn fixtures(num_peers: u16) -> anyhow::Result<Fixtures> {
     match env::var("FM_TEST_DISABLE_MOCKS") {
         Ok(s) if s == "1" => {
             info!("Testing with REAL Bitcoin and Lightning services");
+            let mut config_task_group = TaskGroup::new();
```

Comment:
I'm not an expert here, but I think if we already have a task group we should spawn new ones from it so in case we need to shut everything down we still can:

```
`config_task_group = task_group.make_subgroup()`
```

Reply:
Will fix